pkg/operator/controller/monitoring-dashboard/dashboard.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
I'm wondering what our mid- to longer-term plans are here? Could we encapsulate the metrics in a dashboard-metrics.go (i.e., appropriate types and values) and use `go generate` during the build to generate this JSON file? It would be easier to make larger or more wholesale changes to the output (if necessary).

Reply:
To my knowledge Grafana does not provide tools to generate such dashboard programmatically. Third party tools libraries exist but we did not test them yet.

Please note that this file contain metrics but also many details about the dashboard presentation (Section, subsection, graph width...)

Reply:
There are tools like https://github.com/grafana/grafonnet to generate dashboards programmatically but from my personal experience, nothing really satisfying. Especially if you don't plan to reuse/share the dashboards with other projects, there's little incentive doing so.